src/internal/operators/publish.ts

```diff
@@ -14,14 +14,42 @@ export function publish<T>(selector: MonoTypeOperatorFunction<T>): MonoTypeOpera
  * Returns a ConnectableObservable, which is a variety of Observable that waits until its connect method is called
  * before it begins emitting items to those Observers that have subscribed to it.
  *
+ * <span class="informal">Share the same source, prevent it from emitting values until calling connect.</span>
```

Comment:
@benlesh  Because the example you suggest, I think this does not longer make sense. should I delete it?